jscomp/runtime/js.ml

```diff
@@ -94,6 +94,9 @@ external log3 : 'a -> 'b -> 'c -> unit = "log"
 external log4 : 'a -> 'b -> 'c -> 'd -> unit = "log" 
 [@@bs.val] [@@bs.scope "console"]
 (** A convenience function to log *)
+external logN : 'a array -> unit = "log"
+[@@bs.val] [@@bs.scope "console"] [@@bs.splice]
+(** A convenience function to log a homogeneous collection *)
```

Comment:
Should probably change this to "Log more than 4 items at once" or something. It's not actually logging a collection, since the array turns into a variadic call